clang-tools-extra/clang-tidy/llvmlibc/CalleeNamespaceCheck.cpp

```diff
@@ -45,18 +45,20 @@ void CalleeNamespaceCheck::check(const MatchFinder::MatchResult &Result) {
   if (FuncDecl->getBuiltinID() != 0)
     return;
 
-  // If the outermost namespace of the function is __llvm_libc, we're good.
+  // If the outermost namespace of the function starts with __llvm_libc, we're
+  // good.
   const auto *NS = dyn_cast<NamespaceDecl>(getOutermostNamespace(FuncDecl));
-  if (NS && NS->getName() == "__llvm_libc")
+  if (NS && NS->getName().starts_with("__llvm_libc"))
```

Comment:
Missing test for this change, anyway why this is changed ?

Reply:
The change is to make sure the clang-tidy check works on the new `LIBC_NAMESPACE`. I've added a check to make sure it comes from the macro. I've also added tests for this, and updated the existing tests to handle the changes to this check.